test/image/mocks/box_horz_notched.json

```diff
@@ -0,0 +1,287 @@
+{
+   "data":[
```

Comment:
Very nice test images! re: the labels getting cut off - that'll get fixed by https://github.com/plotly/plotly.js/pull/2243 but here we might as well just give it a larger left margin so they fit.